### PR TITLE
Add voltage source output

### PR DESCRIPTION
**Pull request by couger01**
* Fix auto_range logic

### CODE REVIEW

File: doc/version-history.rst [hunk 1]
```diff
@@ -16,6 +16,8 @@ v0.5.0
 * Add isort suport
 * Add try-except to connect method
 * Add try-except to revert detailed state if command failed
+* Add voltage source command and event
+* Add auto_range active at  logic to set_range command
```
Comment (wvreeven) [inline doc/version-history.rst:20]: `at  logic` -> `at logic` (remove superfluous space).

File: python/lsst/ts/electrometer/controller.py [hunk 2]
```diff
@@ -415,13 +419,14 @@ def parse_buffer(self, response):
         while i < 50000:
             intensity.append(unsorted_values[i])
             time.append(unsorted_values[i + 1])
+            voltage.append(unsorted_values[i + 2])
```
Comment (wvreeven) [inline python/lsst/ts/electrometer/controller.py:422]: Interesting that the variable is called `unsorted_values` but you assume that they actually are sorted. Maybe modify the name of the variable to, for instance, `telemetry`?

**Review by wvreeven — APPROVED** [review]
Looks good to me. I left a few minor comments for you to consider.

File: python/lsst/ts/electrometer/controller.py (hunk 2 above)
Comment (couger01) [reply python/lsst/ts/electrometer/controller.py:422]: It was an old variable name from the code I inherited from the previous developer. I'll change it to raw_values.

**Pull request merged**